pkg/cli/stream/url_to_unstructured_resource_stream_test.go

```diff
@@ -25,7 +25,7 @@ import (
 )
 
 const (
-	spannerDatabaseURLGoldenFile = "testdata/expected-spanner-datbase-url-to-unstructured-resource-stream.golden.yaml"
+	spannerDatabaseURLGoldenFile = "testdata/expected-spanner-database-url-to-unstructured-resource-stream.golden.yaml"
```

Comment:
ah! Good catch. Could you rename the actual file? otherwise we cannot merge this PR.

Reply:
It is renamed https://github.com/GoogleCloudPlatform/k8s-config-connector/commit/ff56ea61f5cc689212c5efa78f2cc29f5be560b1